### PR TITLE
Register and tag CRlibm.jl

### PR DESCRIPTION
**Pull request by dpsanders**
This registers the package `CRlibm.jl`, a wrapper around the `CRlibm` library for correctly-rounded
mathematical functions. See the package `README` for details.

Repo:
https://github.com/dpsanders/CRlibm.jl

Cc @simonbyrne

### CODE REVIEW

**Comment by simonbyrne** [comment]
Great idea!

**Comment by dpsanders** [comment]
Thanks!

**Comment by dpsanders** [comment]
What's going on with the travis failure? I did everything via Pkg.tag and Pkg.submit

**Comment by pao** [comment]
From the Travis output:

```
ERROR: assertion failed: Invalid url scheme http for package CRlibm. Should be 'git' 
 in error at error.jl:21
 in anonymous at no file:15
 in include at ./boot.jl:245
 in include_from_node1 at loading.jl:128
 in process_options at ./client.jl:285
 in _start at ./client.jl:354

while loading /home/travis/build/JuliaLang/METADATA/.test/METADATA.jl, in expression starting on line 4
```

File: CRlibm/url [hunk 1]
```diff
@@ -0,0 +1 @@
+http://github.com/dpsanders/CRlibm.jl
```
Comment (pao) [inline CRlibm/url]: git://github.com/dpsanders.CRlibm.jl.git

**Comment by dpsanders** [comment]
@pao I saw that, thanks, I just don't understand it... What do I need to change where?

**Comment by dpsanders** [comment]
OK, I see that I need to change the URL. Since everything was automatic via `Pkg`, I just don't know where to change it!

**Comment by pao** [comment]
Yeah, if you bootstrapped via Pkg, and still have the steps you followed handy--there might be a Pkg bug in there. If so then steps to reproduce should probably get reported as an issue to the main Julia repository. But you can change the URL manually in the `url` file under your `METADATA.jl` checkout (which I'm not sure what the trivial way to find is these days).

**Comment by mbauman** [comment]
It looks like Pkg.register uses the `origin` remote for its url.  What does `Pkg.Git.run(`remote -v`, dir=Pkg.dir("CRlibm"))` output?  You might have some wonky remotes setup.

**Comment by dpsanders** [comment]
Thanks for the suggestions. I changed the URL by hand in the Pull Request, since I don't have access to the machine that I sent the PR from right now.

I have a feeling that what happened is that I sent the PR from a machine to which I had `Pkg.clone()`d the package from GitHub. Could that be the problem? In any case, it shouldn't be too hard to automatically convert the `http` address to `git` in `Pkg`.

**Comment by tkelman** [comment]
There's a bit of an unresolved issue where pkg should probably be transitioned to use https instead of git, but for now it's good to just keep things uniform.

**Comment by ViralBShah** [comment]
Nice!

**Comment by dpsanders** [comment]
Is there any objection to merging this?

**Comment by dpsanders** [comment]
The problem with the remote URL seems to have been caused by using `hub` to clone the remote, rather  than the `https` line that the clone box on GitHub gives.

**Comment by dpsanders** [comment]
Closing in favor of #3466

**Pull request closed without merging**